src/test/README.md

```diff
@@ -74,3 +74,20 @@ start debugging, just like you would with any other program:
 ```bash
 gdb src/test/test_bitcoin
 ```
+
+#### Segmentation faults
+
+By default, the boost test runner will intercept system errors and not produce a core
+file. To bypass this, add `--catch_system_errors=no` to the `test_bitcoin` arguments
+and ensure that your ulimits are set properly (e.g. `ulimit -c unlimited`). 
+
+Running the tests and hitting a segmentation fault should now produce a file called `core`
+(depending upon your `/proc/sys/kernel/core_pattern` contents).
```

Comment:
Non-blocking nit: Could clarify that ``/proc/sys/kernel/core_pattern` is Linux specific :)

Reply:
Fixed, thanks